jormungandr-watchdog/examples/stdin_echo.rs

```diff
@@ -83,20 +90,109 @@ impl Service for StdoutWriter {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
I made a wrapper so it was easy to implement `Serialize` and `Deserialize`. Couldn't find the **serde** implementation for the `tracing` values.

Reply:
have a look at : https://serde.rs/remote-derive.html